extraction_tools_cache/extract_attackable_npcs.py

```diff
@@ -35,19 +35,13 @@
 from extraction_tools_cache import osrs_cache_data
 
 
-if __name__ == "__main__":
-    import argparse
-    ap = argparse.ArgumentParser()
-    ap.add_argument("-c",
-                    "--cache",
-                    required=True,
-                    help="The compressed cache definitions file to process (npcs.json).")
-    args = vars(ap.parse_args())
-    path_to_compressed_json_file = args["cache"]
+def main(compressed_json_file_path: str):
+    """Main function to extract attackble NPC definition files
 
+    :param compressed_json_file_path: Compressed cache file
+    """
     attackable_npcs = {}
-
-    compressed_json_file = pathlib.Path() / path_to_compressed_json_file
+    compressed_json_file = pathlib.Path() / compressed_json_file_path
```

Comment:
You should be able to just put this into a `Path`, without the need for `/` operator.
```
>>> from pathlib import Path
>>> Path("my/path")
PosixPath('my/path')
>>> Path() / "my/path"
PosixPath('my/path')
```